### ISSUE TITLE
React router migration

### ISSUE THREAD

**Issue by ltdonner** [loblaw-sre/backstage-plugin-gitlab#213]
This plugin was one that is showing react-router@6.0.0-beta.0

Requesting this plugin be migrated so that it will work with the 1.6 version of Backstage with the React Router 6.0 migration

https://backstage.io/docs/tutorials/react-router-stable-migration#step-3---ensure-that-your-external-plugins-are-updated

**Comment by antoniomuso** [comment]
Yes, this plugin uses an old version of `@backstage/core-components`. In this old version, we already have as dependency  `react-router@6.0.0-beta.0`.

**Comment by jm-ra** [comment]
Any tips on a workaround?

**Comment by ivangonzalezacuna** [comment]
In our case, we just deactivated the plugin temporarily until this PR is merged: https://github.com/loblaw-sre/backstage-plugin-gitlab/pull/190. 

It was blocking us to fully upgrade to the latest version of backstage, so maybe this is the best workaround for now. Or just create a fork from the code in the linked PR, which works with the latest version of react-router.

**Comment by testn** [comment]
Can we get this sort out?
